jsconfig.json

```
@@ -5,7 +5,7 @@
 		"target": "ES6",
 		"baseUrl": ".",
 		"paths": {
-			"*": [ "test/*", "server/*", "client/*" ]
+			"*": [ "test/*", "client/*" ]
```

Can't `server` paths still be accessed like a root?

```suggestion
			"*": [ "test/*", "client/*", "client/server/*" ]
```

---

@sirreal I don't think so? What makes you think they still can be?

---

I wasn't aware this had changed, seems like you're right 👍

---

Thanks for checking and confirming 👍